tests/tester.js

```diff
@@ -42,6 +47,10 @@ if (argv.r) {
   runAll()
 }
 
+function skipTest (testName) {
+  return skip.map((skipName) => (new RegExp(`^${skipName}`)).test(testName)).some(isMatch => isMatch)
+}
+
```

Comment:
I guess that'd be neater. The array of test names `skip` would be passed instead of a function `skipFn`.